.NET/Microsoft.Recognizers.Text.sln

```diff
@@ -1,168 +1,169 @@
-﻿
-Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Nitpick: Be careful to not commit files where the only changes are line break differences (\n vs \r\n). It makes reviewing more complicated than necessary.